xdc-plugin/xdc.cc

```diff
@@ -345,10 +294,18 @@ struct SetProperty : public Pass {
 
 	// Extract signal name and port bit information from port name
 	std::pair<std::string, int> extract_signal(const std::string& port_name) {
+	        std::regex port_regex("\\{?([\\\\a-zA-Z0-9_\\.]+)(\\[([0-9]+)\\])?.*");
```

Comment:
Does this match the underlying verilog/VHDL spec for ports?  This seems more restrictive than before

Reply:
This was a leftover from an experimental implementation. Removed.